README.md

```diff
@@ -39,6 +39,10 @@ pip install --upgrade diffusers  # should install diffusers 0.2.4
 conda install -c conda-forge diffusers
 ```
 
+**Apple Silicon (M1/M2) support**
+
+See [README_mps.md](README_mps.md).
+
```

Comment:
Think we should move this under docs or examples. Maybe it makes sense now to add a doc section now that explains how to run diffusers with mps, onnx, and other hardwares. cc @anton-l @patrickvonplaten

Reply:
Yeah I wasn't sure where to put it. `docs` sounds good to me, should we create a new section there?

Reply:
yeah a new section called `optimization` where we could put docs related to mps, onnx, openvino and AMD support etc.